modules/cli/cmd/devrunner/config/networks.go

```diff
@@ -4,194 +4,140 @@ import (
 type NetworkConfig struct {
-	SequencerChainId string `mapstructure:"sequencer_chain_id" toml:"sequencer_chain_id"`
-	SequencerGRPC    string `mapstructure:"sequencer_grpc" toml:"sequencer_grpc"`
-	SequencerRPC     string `mapstructure:"sequencer_rpc" toml:"sequencer_rpc"`
-	RollupName       string `mapstructure:"rollup_name" toml:"rollup_name"`
-	NativeDenom      string `mapstructure:"default_denom" toml:"default_denom"`
+	SequencerChainId string                   `mapstructure:"sequencer_chain_id" toml:"sequencer_chain_id"`
+	SequencerGRPC    string                   `mapstructure:"sequencer_grpc" toml:"sequencer_grpc"`
+	SequencerRPC     string                   `mapstructure:"sequencer_rpc" toml:"sequencer_rpc"`
+	RollupName       string                   `mapstructure:"rollup_name" toml:"rollup_name"`
+	NativeDenom      string                   `mapstructure:"default_denom" toml:"default_denom"`
+	Services         map[string]ServiceConfig `mapstructure:"services" toml:"services"`
 }
 
 // DefaultNetworksConfigs returns a NetworksConfig struct populated with all
 // network defaults.
-func DefaultNetworksConfigs() NetworkConfigs {
-	config := NetworkConfigs{
-		Local: NetworkConfig{
-			SequencerChainId: "sequencer-test-chain-0",
-			SequencerGRPC:    "http://127.0.0.1:8080",
-			SequencerRPC:     "http://127.0.0.1:26657",
-			RollupName:       "astria-test-chain",
-			NativeDenom:      "nria",
-		},
-		Dusk: NetworkConfig{
-			SequencerChainId: "astria-dusk-" + duskNum,
-			SequencerGRPC:    "https://grpc.sequencer.dusk-" + duskNum + ".devnet.astria.org/",
-			SequencerRPC:     "https://rpc.sequencer.dusk-" + duskNum + ".devnet.astria.org/",
-			RollupName:       "",
-			NativeDenom:      "nria",
-		},
-		Dawn: NetworkConfig{
-			SequencerChainId: "astria-dawn-" + dawnNum,
-			SequencerGRPC:    "https://grpc.sequencer.dawn-" + dawnNum + ".devnet.astria.org/",
-			SequencerRPC:     "https://rpc.sequencer.dawn-" + dawnNum + ".devnet.astria.org/",
-			RollupName:       "",
-			NativeDenom:      "ibc/channel0/utia",
-		},
-		Mainnet: NetworkConfig{
-			SequencerChainId: "astria",
-			SequencerGRPC:    "https://grpc.sequencer.astria.org/",
-			SequencerRPC:     "https://rpc.sequencer.astria.org/",
-			RollupName:       "",
-			NativeDenom:      "ibc/channel0/utia",
+func DefaultNetworksConfigs(defaultBinDir string) NetworkConfigs {
+	return NetworkConfigs{
+		Configs: map[string]NetworkConfig{
+			"local": {
+				SequencerChainId: "sequencer-test-chain-0",
+				SequencerGRPC:    "http://127.0.0.1:8080",
+				SequencerRPC:     "http://127.0.0.1:26657",
+				RollupName:       "astria-test-chain",
+				NativeDenom:      "nria",
+				Services: map[string]ServiceConfig{
+					"conductor": {
+						Name:        "astria-conductor",
+						Version:     "v" + AstriaConductorVersion,
+						DownloadURL: findBinaryByName(Binaries, "astria-conductor").Url,
+						LocalPath:   filepath.Join(defaultBinDir, "astria-conductor-v"+AstriaConductorVersion),
```

Comment:
```suggestion
						LocalPath:   filepath.Join(defaultBinDir, "astria-conductor-v" + AstriaConductorVersion),
```
Let's be consistent with spacing around the + operator. Unfortunately gofmt doesn't take a strong stance on it, but I prefer spaces between operators and we have been using spaces in this codebase so far I think

Reply:
Disregard. Looks like this is correct formatting for function parameters!
https://github.com/golang/go/issues/34426